src/test/java/fi/otavanopisto/kuntaapi/test/server/integration/casem/CasemPageTestsIT.java

```diff
@@ -0,0 +1,199 @@
+      .then()
+      .assertThat()
+      .statusCode(200)
+      .body("id.size()", is(4))
+      .body("meta[0].hideMenuChildren", is(true))
+      .body("meta[1].hideMenuChildren", is(true))
+      .body("meta[2].hideMenuChildren", is(true))
+      .body("meta[3].hideMenuChildren", is(true));
+  } 
+  
+  @Test
+  public void testListPagesByPath() {
+    given() 
+      .baseUri(getApiBasePath())
+      .contentType(ContentType.JSON)
+      .get("/organizations/{organizationId}/pages?path=/test_board", getOrganizationId(0))
+      .then()
+      .assertThat()
+      .statusCode(200)
+      .body("id.size()", is(1))
+      .body("id[0]", notNullValue())
+      .body("slug[0]", is("test_board")); 
+      
+    given() 
+      .baseUri(getApiBasePath())
+      .contentType(ContentType.JSON)
+      .get("/organizations/{organizationId}/pages?path=/test_board/meeting_16.1.2017/test_2", getOrganizationId(0))
+      .then()
+      .assertThat()
+      .statusCode(200)
+      .body("id.size()", is(1))
+      .body("id[0]", notNullValue())
+      .body("slug[0]", is("test_2")); 
+    
+    given() 
+      .baseUri(getApiBasePath())
+      .contentType(ContentType.JSON)
+      .get("/organizations/{organizationId}/pages?path=/non-existing", getOrganizationId(0))
+      .then()
+      .assertThat()
+      .statusCode(200)
+      .body("id.size()", is(0));
+    
+    given() 
+      .baseUri(getApiBasePath())
+      .contentType(ContentType.JSON)
+      .get("/organizations/{organizationId}/pages?path=/test_board/non-existing", getOrganizationId(0))
+      .then()
+      .assertThat()
+      .statusCode(200)
+      .body("id.size()", is(0));
+  } 
+  
+  @Test
+  public void testOrganizationPagesNotFound() throws InterruptedException {
+    String organizationId = getOrganizationId(0);
+    String incorrectOrganizationId = getOrganizationId(1);
+    String organizationAnnouncecmentId = getOrganizationPageId(organizationId, 0);
+    
+    String[] malformedIds = new String[] {"evil", "*", "/", "1", "-1", "~"};
+    assertFound(String.format("/organizations/%s/pages/%s", organizationId, organizationAnnouncecmentId));
+    assertEquals(4, countApiList(String.format("/organizations/%s/pages", organizationId)));
+    
+    for (String malformedId : malformedIds) {
+      assertNotFound(String.format("/organizations/%s/pages/%s", organizationId, malformedId));
+    }
+    
+    assertNotFound(String.format("/organizations/%s/pages/%s", incorrectOrganizationId, organizationAnnouncecmentId));
+    assertEquals(0, countApiList(String.format("/organizations/%s/pages", incorrectOrganizationId)));
+  }
+  
+  private void createPtvSettings() {
+    insertSystemSetting(PtvConsts.SYSTEM_SETTING_BASEURL, String.format("%s%s", getWireMockBasePath(), BASE_URL));
+    flushCache();
+  }
+  
+  private void deletePtvSettings() {
+    deleteSystemSetting(PtvConsts.SYSTEM_SETTING_BASEURL);
+  }
+    
+  private void createCasemSettings(String organizationId) {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Remove this unused private "createCasemSettings" method. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=squid%3AUnusedPrivateMethod)